Add SSL Option to Backend Service (#1011)
kyma-project/busola: docs/install-kyma-dashboard-manually.md

3. Wait a minute for the Dashboard to start.
4. The Dashboard is available under your domain (for example, `https://busola.local.kyma.dev/`).

## Enable TLS in the backend service

> By default, the communication with the backend pod is plain HTTP. Even when you use istio service mesh, the last bit of communication

Reviewer: ```suggestion
By default, the communication with the backend Pod is plain HTTP. Even when you use Istio Service Mesh, the last bit of communication between the Istio sidecar and the backend service stays plain HTTP.
```

Capitalization as in [Kyma style guide](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalized-terms).
Let's not split a sentence in two lines.

> between the istio sidecar and the backend service stays plain HTTP.

Reviewer: ```suggestion
```

> If your use-case requires TLS all the way into the backend pod, you can enable it by providing a TLS certificate to the backend service,

Reviewer: ```suggestion
If your use case needs TLS all the way into the backend Pod, enable it by providing a TLS certificate to the backend service, setting some environment variables and changing the deployment.
```

> setting some environment variables and changing the deployment.

Reviewer: ```suggestion
```

> 

Reviewer: Are there any prerequisites to these instructions?

> 1. Generate TLS certificate

Reviewer: ```suggestion
1. Generate a TLS certificate.
```

Is any TLS certificate good, or does it have to fulfil any special requirements regarding encoding etc?

> 2. Mount your certificate into the backend pod

Reviewer: ```suggestion
2. Mount your certificate into the backend Pod.
```

> 3. Set the following environment variables on the backend pod:

Reviewer: ```suggestion
3. Set the following environment variables on the backend Pod:
```

   ```
   BUSOLA_SSL_ENABLED=1
   BUSOLA_SSL_KEY_FILE=/path/to/tls.key
   BUSOLA_SSL_CRT_FILE=/path/to/tls.crt
   ```

> 4. If you are using istio service mesh, enable the `destinationrule-busola-backend.yaml` resource in `resources/istio/kustomization.yaml`

Reviewer: ```suggestion
4. If you are using Istio Service Mesh, enable the `destinationrule-busola-backend.yaml` resource in `resources/istio/kustomization.yaml`.
```

> 5. Install Busola according to [Install Kyma Dashboard with Istio Ingress](#install-kyma-dashboard-with-istio-ingress)

Reviewer: ```suggestion
5. Install Busola according to [Install Kyma Dashboard with Istio Ingress](#install-kyma-dashboard-with-istio-ingress).
```

## Enable TLS in the backend service

By default, the communication with the backend Pod is plain HTTP. Even when you use Istio Service Mesh, the last bit of communication between the Istio sidecar and the backend service stays plain HTTP.

> If your use case needs TLS all the way into the backend Pod, enable it by providing a TLS certificate to the backend service, setting some environment variables and changing the deployment. Depending on your setup, you might have to use a valid TLS certificate (e.g. if you are directly exposing the backend Pod without TLS termination in between). In this case you might want to use cert-manager to handle your certificates or provide a valid one here and manage it yourself.

Reviewer: ```suggestion
If your use case needs TLS all the way into the backend Pod, enable it by providing a TLS certificate to the backend service, setting some environment variables and changing the deployment. Depending on your setup, you might have to use a valid TLS certificate (for example, if you are directly exposing the backend Pod without TLS termination in between). In this case, you might want to use cert-manager to handle your certificates, or provide a valid one here and manage it yourself.
```
